mica/starcheck/starcheck.py

```diff
@@ -20,60 +22,197 @@
 
 DEFAULT_CONFIG = dict(dbi='sqlite',
                       server=os.path.join(MICA_ARCHIVE, 'starcheck', 'starcheck.db3'),
-                      mp_top_level='/data/mpcrit1/mplogs')
+                      mp_top_level='/data/mpcrit1/mplogs',
+                      timelines_db=dict(dbi='sqlite',
+                                        server=os.path.join(os.environ['SKA'], 'data', 'cmd_states', 'cmd_states.db3')))
 FILES = dict(data_root=os.path.join(MICA_ARCHIVE, 'starcheck'),
              touch_file=os.path.join(MICA_ARCHIVE, 'starcheck', "starcheck_parser.touch"),
              sql_def='starcheck.sql')
 
 # this doesn't belong in starcheck
-def get_timeline_at_date(date):
-    aca_db = Ska.DBI.DBI(dbi='sybase', server='sybase', user='aca_read')
-    return aca_db.fetchone(
+def get_timeline_at_date(date, timelines_db=None):
+    date = DateTime(date).date
+    if timelines_db is None:
+        timelines_db = Ska.DBI.DBI(**DEFAULT_CONFIG['timelines_db'])
+    return timelines_db.fetchone(
         "select * from timeline_loads where datestop >= '%s' "
         " and datestart <= '%s' and scs <= 130 order by datestart desc"
         % (date, date))
 
-def get_mp_dir(obsid, config=None):
+
+def get_flickpix_mons(start=None, config=None):
```

Comment:
Also, I was originally going to move this out of here because it is very specific to the task at hand, but on the other... hand, it *is* really just a special star catalog lookup, so maybe it isn't dreadful to just leave it in the star catalog lookup utilities.